### DISCUSSION TITLE
Can I create more then one user?

### DISCUSSION THREAD

**Discussion by UberMetroid** [photoprism/photoprism#742] [Q&A] [answered]
Is "admin" the only user I can create?

I just found this and I am testing it out / learning what it can do.

**Comment by graciousgrey** [comment, marked as answer]
Yes, at the moment there is only one user. 
We plan to implement user management next year: https://github.com/photoprism/photoprism/issues/98 :)

**Comment by aschmitz-GH** [comment]
just saw this amazing app and wondering if this is already implemented.
I am searching for an app to share my pics with family and friends, so I need 1 admin account and a public account (read only)- multi user with ACL would be awesome but not necessary atm.

**Reply by graciousgrey** [reply]
Multi user functionality is still work in progress

#98 

https://github.com/photoprism/photoprism/discussions/1678
